central/imageintegration/service/service_impl.go

```diff
@@ -153,6 +154,11 @@ func (s *serviceImpl) PostImageIntegration(ctx context.Context, request *storage
 		return nil, errors.Wrap(errox.InvalidArgs, "id field should be empty when posting a new image integration")
 	}
 
+	// Do not allow manual creation of a scanner v4 integration.
```

Comment:
Can only comment here, but the comment is meant for the `PutImageIntegration` part: we also have to verify you cannot change the type of an image integration to `scannerV4` nor that you can change the default V4 scanner integration.

Reply:
Good call! Didn't realize we allowed an integration's type to be changed.  Changing values of the default `scannerv4` integration is acceptable, but changing it to a different type or some other integration to `scannerv4` type is not desired.